gdi/get-data-in/rum/browser/configure-rum-browser-instrumentation.rst

```diff
@@ -314,7 +314,7 @@ Traces that happen asynchronously, such as user interactions that result in a pr
 -  ``MessagePort``
 -  Hash-based routers
 
-To activate asynchronous traces, set the ``context.async`` property to ``true``.
+To asynchronous trace linking is enabled by default. In case of compatibility issues you can disable it by setting the ``context.async`` property to ``false``.
```

Comment:
```suggestion
Asynchronous trace linking is activated by default. In case of compatibility issues you can disable it by setting the ``context.async`` property to ``false``.
```